dds/idl/be_global.cpp

```diff
@@ -588,7 +637,7 @@ BE_GlobalData::is_topic_type(AST_Decl* node)
           || type->node_type() == AST_Decl::NT_union) {
         return node->annotations().find(topic_annotation_);
       }
-    }
+    } 
```

Comment:
In the nightly builds, there is a check for trailing whitespace. I use the builtin feature in vim that highlights trailing whitespace in C/C++:

```vim
" C Syntax
let c_gnu=1
" Highlight Trailing Space
let c_space_errors = 1
```

Reply:
Thanks!